test/e2e/kubelet_stats.go

```diff
@@ -692,7 +697,7 @@ func (r *resourceMonitor) Start() {
 	}
 	r.collectors = make(map[string]*resourceCollector, 0)
 	for _, node := range nodes.Items {
-		collector := newResourceCollector(r.client, node.Name, r.containers, pollInterval)
+		collector := newResourceCollector(r.client, node.Name, r.containers, r.pollingInterval)
```

Comment:
Uh...I included this fix in my other PR with the expectation that this PR would land first, but mine got merged first :-( Sorry and could you rebase?